### DISCUSSION TITLE
Is this sdk compatible with php version 5.6 ?

### DISCUSSION THREAD

**Discussion by belly-hostinger** [SkyAPM/SkyAPM-php-sdk#403] [Q&A] [answered]
I only see the Dockerfile only for PHP version 7 & 8

**Comment by heyanlong** [comment, marked as answer]
Does not support PHP5

**Comment by ghost** [comment]
I am starting the PHP agent via docker agent  as below 

docker run --restart always -d -e SW_OAP_ADDRESS=oap:11800 skyapm/skywalking-php

But i am not able to see any data on the UI

**Reply by heyanlong** [reply]
Maybe the oap address is not found in the inside the container.

**Reply by ghost** [reply]
I changed the OAP server address to localhost. Shouldn't it be showing the data related to all PHP applications running ?

**Reply by heyanlong** [reply]
Need to ensure that the oap server can be accessed inside the container
